yt_dlp/extractor/jiocinema.py

```diff
@@ -0,0 +1,324 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+import json
+import math
+import re
+import uuid
+
+
+class JioCinemaBaseIE(InfoExtractor):
+    deviceId = None
+    auth_token = None
+
+    def _get_auth_token(self):
+        token_endpoint = "https://auth-jiocinema.voot.com/tokenservice/apis/v4/guest"
+
+        headers = {
+            'authority': 'auth-jiocinema.voot.com',
+            'accept': 'application/json, text/plain, */*',
+            'accept-language': 'en-IN,en;q=0.9',
+            'content-type': 'application/json',
+            'dnt': '1',
+            'origin': 'https://www.jiocinema.com',
+            'referer': 'https://www.jiocinema.com/',
+            'sec-ch-ua': '"Google Chrome";v="119", "Chromium";v="119", "Not?A_Brand";v="24"',
+            'sec-ch-ua-mobile': '?0',
+            'sec-ch-ua-platform': '"macOS"',
+            'sec-fetch-dest': 'empty',
+            'sec-fetch-mode': 'cors',
+            'sec-fetch-site': 'cross-site',
+            'user-agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/119.0.0.0 Safari/537.36'
+        }
```

Comment:
some of these seem unnecessary, though can't tell /shrug

Reply:
Fixed